drivers/pds/lib/utils.go

```diff
@@ -125,7 +125,7 @@ const (
 	appConfigTemplateName = "QaDefault"
 	defaultRetryInterval  = 10 * time.Minute
 	duration              = 900
-	timeOut               = 5 * time.Minute
+	timeOut               = 15 * time.Minute
```

Comment:
5min timeOut is being used by other test cases and it's expected to fail if desired result is not achieved in 5min, can we have a new var for 15min timeOut?

Reply:
This is the default timeout time which is implicitly used in all the functions and this should be implemented as an explicit param for  ValidateDataServiceDeployment func().
Changing this will require a change in every instance where it has been called.

Also, For the rolling reboot tests, it was failing to come up in 10 minutes, also I have noticed 10 node clusters used to take more than 5 minutes in general. I am not sure if that has been tested so far through automation.

Hence keeping it as it is for now and as part of enhancement, I will modify this to make the timeout period explicit.

Please share your thoughts , if required i can revert it till the enhancement will be done.

Reply:
Got it, This test case needs more than 10min for the deployment pods to come up, its better to make timeout as an explicit param. 
Kindly add a TODO comment and create a task ticket for the same.

Reply:
Jira ticket for ref: https://portworx.atlassian.net/browse/PA-401